apps/api/Dockerfile.dev

```diff
@@ -0,0 +1,7 @@
+FROM node:14.17.1
+
```

Comment:
Filename Dockerfile should be fine.

Reply:
I meant for `.dev` to be a marker that this isn't a typical dockerfile and isn't meant to be used for deployments

Reply:
I agree as this is is appropriate where we deploy with Docker. In this case, we don't have a prod Dockerfile so, just Sockerfile should be okay